### ISSUE TITLE
Modification in UI advanced example

### ISSUE THREAD

**Issue by arifroni** [DeebotUniverse/docs#75]
Hello

I noticed I need to do a small modification in the [advanced example](https://deebot.readthedocs.io/integrations/home-assistant/examples/ui/advanced/).

to get the rooms:
`{%- set data.rooms = data.rooms + [rooms[room_name]] -%}`
should be 
`{%- set data.rooms = data.rooms + [rooms[room_name[5:]]] -%}`

BTW, this is required if your room name doesn't start with "room_".

**Comment by edenhaus** [comment]
I quickly checked it and the example works as expected:
![image](https://user-images.githubusercontent.com/26537646/142843153-f2a3aebc-2777-44b6-bf00-52eda7f4572b.png)

Can you please paste the following snippet in your developer template view?
```jinja2
{%- set queue = "variable.deebot_susi_queue" -%}
{%- set vacuum_bot = "vacuum.susi" -%}

{%- set queue_split = states(queue).split(",") -%}
{%- set rooms = state_attr(vacuum_bot, "rooms")-%}
{%- set data = namespace(rooms=[]) -%}
{%- for room_name in queue_split -%}
  {%- set data.rooms = data.rooms + [rooms[room_name]] -%}
{%- endfor -%}
{{ data.rooms | join(",") }}

For debugging:
rooms: {{ rooms }}
queue_split: {{ queue_split }}
```

Which version of the custom integration are you using?

**Comment by arifroni** [comment]
![image](https://user-images.githubusercontent.com/59896374/142847349-0a7ca8d2-b6c1-4e59-80af-4127e0fd5150.png)

i am using the following:
![image](https://user-images.githubusercontent.com/59896374/142847461-c706002f-64a0-4214-a6fe-ff568d5a7c32.png)

**Comment by arifroni** [comment]
I think understand the problem now. I am adding room_* in the queue when I press the button for a specific room. maybe it was needed in the previous integration. if I remove that then the modification is not needed anymore.

![image](https://user-images.githubusercontent.com/59896374/142847974-b1b5d7aa-1ffc-4a88-83c9-b27b2288a430.png)

**Comment by edenhaus** [comment]
Yeah you need to change it after the migration. Thanks for pointing this out, I will add a note on the migration guide

**Comment by edenhaus** [comment]
Fixed in #76
